tsg/config.py

```diff
@@ -21,6 +21,7 @@
 os.makedirs(INTERMEDIATE_DIR, exist_ok=True)
 
 FIELDS = ['type', 'title', 'isbn', 'content']
+FIELD_WEIGHTS = [2.5, 2.5, 5, 1]
```

Comment:
This is by accident. I thought for both values 2.5 is appropriate..

Brian A. Ignacio <notifications@github.com> schrieb am Mi., 20. Apr. 2016,
23:15:

> In tsg/config.py
> <https://github.com/moritzschaefer/the-search-engine/pull/32#discussion_r60427126>
> :
>
> > @@ -21,6 +21,7 @@
> >  os.makedirs(INTERMEDIATE_DIR, exist_ok=True)
> >
> >  FIELDS = ['type', 'title', 'isbn', 'content']
> > +FIELD_WEIGHTS = [2.5, 2.5, 5, 1]
>
> Type and title has the same weight? Can you tell us why ?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/moritzschaefer/the-search-engine/pull/32/files/e66a2d8e8a9a462ad46caa6bdbc8e847be8ae108#r60427126>
>